### ISSUE TITLE
Training crashes after some steps.

### ISSUE THREAD

**Issue by Sreerag-ibtl** [RomRoc/objdet_train_tensorflow_colab#6]
First of all let me say this is not an issue regarding your code, but I think you will have some suggestions to make. I am training custom object detector with ssd_mobilenet_v2_coco as the pre-trained model. For this I am following this tutorial : https://tensorflow-object-detection-api-tutorial.readthedocs.io/en/latest/index.html
But in the training section it crashed after about 700 steps. My train set has about 2000 images and test set has 50. The batch size is 24, which is the default value in the config file.
Have you faced any similar issues when training? Would like to hear your suggestions.
